pkg/webhook/server/webhook_cert_init.go

```diff
@@ -83,13 +94,17 @@ func syncConfigMap(ctx context.Context, crclient client.Client, namespace string
 		if !apierrors.IsAlreadyExists(err) {
 			return err
 		}
-		existingCfg := &v1.Service{}
+		existingCfg := &corev1.ConfigMap{}
 		err := crclient.Get(ctx, types.NamespacedName{
 			Name:      secureConfigMap.Name,
 			Namespace: secureConfigMap.Namespace,
 		}, existingCfg)
-
-		err = crclient.Update(ctx, secureConfigMap)
+		if err != nil {
+			return err
+		}
+		existingCfg.Data = secureConfigMap.Data
```

Comment:
secureConfigMap data are always empty. 
The main configuration secureConfigMap should have - `"service.beta.openshift.io/inject-cabundle": "true"` annotation.
So, I think here it makes sense port resourceVersion to secureConfigMap, and update with new object.

Reply:
Fixed to copy only labels/annotations to existing cfg.